python/tskit/tables.py

```diff
@@ -4334,6 +4334,8 @@ def union(
         self,
         other,
         node_mapping,
+        all_edges=False,
```

Comment:
Placing these at the start of kwargs has broken compatibility with old code that might have been using them positionally as we had no `*`.

Reply:
Oh, good point. Easy enough to put a `*` in now, right?

Reply:
Yeah, add a star, but just before your new kwargs, to not break existing positional usage.